pkg/appconsts/v1/app_consts.go

```diff
@@ -0,0 +1,7 @@
+package v1
+
+const (
+	Version              = 1
+	MaxSquareSize        = 128
+	SubtreeRootThreshold = MaxSquareSize
```

Comment:
just flagging this will have merge conflicts with https://github.com/celestiaorg/celestia-app/pull/1766

Reply:
Yeah I'm aware of that. It will also conflict with another PR that renamed DefaultMaxSquareSize to MaxSquareSize